src/snowflake/snowpark/_internal/analyzer/analyzer_utils.py

```diff
@@ -752,6 +757,14 @@ def get_comment_sql(comment: Optional[str]) -> str:
     )
 
 
+def get_assign_param_sql(param_name: str, param_value: Optional[Any]) -> str:
+    return (
+        f"{param_name}{EQUALS}{param_value}"
+        if param_value is not None
+        else EMPTY_STRING
+    )
```

Comment:
Could `get_options_statement` be used in place of adding a new function?

Reply:
that's for pointing this out. I didn't know this existed